### ISSUE TITLE
Test for WCAG failure F37: Launching a new window without prior warning when the status of a radio button, check box or select list is changed

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#192]
http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/F37
This document describes a failure that occurs when changing the status of a radio button, a check box or an item in a select list causes a new window to open. It is possible to use scripting to create an input element that causes a change of context (submit the form, open a new page, a new window) when the element is selected. Developers can instead use a submit button (see G80: Providing a submit button to initiate a change of context) or clearly indicate the expected action. 
## Test procedure
1. Find each form in a page.
2. For each form control that is a radio button, check box or an item in a select list, check if changing the status of the control launches a new window.
3. For each new window resulting from step 2, check if the user is warned in advance.
